### PR TITLE
Add rules

### PR DESCRIPTION
**Pull request by adamrysyk**
add a rules presentational component and change the flow when starting a new game

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { SHOW_QUESTION, QUESTION_ANSWERED, HIDE_INTRO } from '../constants';
+import { SHOW_QUESTION, QUESTION_ANSWERED, CONFIRM_INTRO, CONFIRM_RULES } from '../constants';
```
Comment (phi-tran-vibe) [inline src/actions/index.js]: these should be called `CONTINUE_` instead of `CONFIRM_`

File: src/actions/index.js [hunk 2]
```diff
@@ -15,8 +15,14 @@ export const questionAnswered = ({ questionID, optionID }) => {
     };
 };
 
-export const hideIntro = () => {
+export const confirmIntro = () => {
```
Comment (phi-tran-vibe) [inline src/actions/index.js]: these function names too

File: src/actions/index.js (hunk 1 above)
Comment (adamrysyk) [reply src/actions/index.js]: done

File: src/actions/index.js (hunk 2 above)
Comment (adamrysyk) [reply src/actions/index.js]: done

**Pull request merged**